api/facadeversions.go

```diff
@@ -48,7 +48,7 @@ var facadeVersions = facades.FacadeVersions{
 	"Cleaner":                      {2},
 	"Client":                       {6, 7},
 	"Cloud":                        {7},
-	"Controller":                   {11, 12},
+	"Controller":                   {12},
```

Comment:
@SimonRichardson do I need to do this?

Reply:
100%. Yeah, we're stating that we're dropping support for version 11, with this API client.

Reply:
We'll need to tell the JAAS/JIMM guys that this is getting deprecated. Can you add it to the risk register, so we can tell them this and many other changes are likely to happen when 4.0 is released. Thanks.

Reply:
added to risk register